libraries/controllers/src/controllers/UserInputMapper.cpp

```diff
@@ -1177,13 +1181,22 @@ Mapping::Pointer UserInputMapper::parseMapping(const QString& json) {
     if (doc.isNull()) {
         qCDebug(controllers) << "Invalid JSON...\n";
         qCDebug(controllers) << error.errorString();
+#if (QT_VERSION < QT_VERSION_CHECK(5, 14, 0))
+        qCDebug(controllers) << "JSON was:\n" << json << endl;
```

Comment:
I think we could do something like:

     #if (QT_VERSION < QT_VERSION_CHECK(5, 14, 0))
         #define VEndl endl
     #else
         #define VEndl Qt::endl
     #endif